source/code/python/authentication/mfa_add_ga_factor_req.py

```diff
@@ -0,0 +1,5 @@
+google_factor = account.factors.create({
+    'type': 'google-authenticator',
+    'accountName': 'example@stormpath.com',
+    'issuer': 'myEnterpriseIssuer'}
+)
```

Comment:
Can we get the closing curly brace on the bottom line here? This way it looks nicer.